Source/Core/Common/CRC32.cpp

```diff
@@ -0,0 +1,15 @@
+// Copyright 2021 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <zlib.h>
+
+#include "Common/CRC32.h"
+
+u32 ComputeCRC32(const std::string& data)
```

Comment:
```suggestion
u32 ComputeCRC32(std::string_view data)
```

Would allow this to be used in a non-allocating manner as well.

Reply:
Thanks, fixed!